### ISSUE TITLE
More control on Cache permissions

### ISSUE THREAD

**Issue by stuartcusackie** [statamic/ideas#815]
I'd like to be able to give my content managers the ability to wipe the static cache / stache / application cache, but I don't want them touching the glide cache.

Reason being is that clearing the glide cache can crash the website when you have lots of images with responsive sizes and jpeg fallbacks. For example, on my current project, 100+ images for a single page load would need to be regenerated by glide if the cache was cleared. Small cloud instances get easily overloaded by these requests.

I would still like my managers to be able to clear the static cache / stache in case of cached content bugs.

Thanks!

**Comment by edalzell** [comment]
The glide cache isn't in the stache nor Laravel's cache, is it?

Pretty sure it's separate.

**Comment by duncanmcclean** [comment]
> The glide cache isn't in the stache nor Laravel's cache, is it?
> 
> Pretty sure it's separate.

Yes the caches themselves are separate but the permissions for the utility aren’t.
